numpy/random/tests/test_randomstate.py

```diff
@@ -364,20 +364,20 @@ def test_repeatability(self):
                'uint8':  '001aac3a5acb935a9b186cbe14a1ca064b8bb2dd0b045d48abeacf74d0203404'}  # noqa: E501
 
         for dt in self.itype[1:]:
-            random.seed(1234)
+            rng = random.RandomState(1234)
 
             # view as little endian for hash
             if sys.byteorder == 'little':
-                val = self.rfunc(0, 6, size=1000, dtype=dt)
+                val = rng.randint(0, 6, size=1000, dtype=dt)
```

Comment:
There are more `self.rfunc` references in this class that still need to be removed. Delete the class attribute as it's no longer necessary.